factory-reset/eos-factory-reset-users.service

```diff
@@ -15,12 +15,16 @@
 # unprivileged users will login before it finishes, as a tmpfile snippet
 # creates /run/nologin during early boot and systemd-user-sessions.service
 # removes it when the boot is finished. See pam_nologin(8) for more info.
+#
+# This unit also needs to be ordered before display-manager.service, to have
+# GDM started only after all users are already removed so it knows it should
+# launch the FBE.
 
 [Unit]
 Description=Remove All Users And Home Directories
 RefuseManualStart=yes
 Wants=dbus.service
-Before=systemd-user-sessions.service
+Before=systemd-user-sessions.service display-manager.service
```

Comment:
I don't see how this will have any effect because gdm.service has:

```ini
After=rc-local.service plymouth-start.service systemd-user-sessions.service
#                                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

so it is already (transitively) ordered before GDM.

Reply:
Yes, you are right, now that you mentioned I remember realizing this when it was originally implemented. Closing this PR.